images/win/scripts/Installers/Install-Haskell.ps1

```diff
@@ -45,4 +17,27 @@ Set-SystemVariable "CABAL_DIR" $cabalDir
 Add-MachinePathItem "$ghcupPrefix\ghcup\bin"
 Add-MachinePathItem "$cabalDir\bin"
 
+Update-SessionEnvironment
+
+# Get 3 latest versions of GHC
+$Versions = ghcup list -t ghc -r | Where-Object {$_ -notlike "prerelease"}
+$VersionsOutput = [Version[]]($Versions | %{ $_.Split(' ')[1]; })
+$LatestMajorMinor = $VersionsOutput | Group-Object { $_.ToString(2) } | Sort-Object { [Version]$_.Name } | Select-Object -last 3
+$VersionsList = $LatestMajorMinor | ForEach-Object { $_.Group | Select-Object -Last 1 } | Sort-Object
+
+# The latest version will be installed as a default
+ForEach ($version in $VersionsList)
+{
+    Write-Host "Installing ghc $version..."
+    ghcup install ghc $version
+    ghcup set ghc $version
+}
+
+# Add default version of GHC to path
+$DefaultGhcVersion = $VersionsList | Select-Object -Last 1
+ghcup set ghc $DefaultGhcVersion
+
+Write-Host 'Installing cabal...'
+ghcup install cabal 3.8.1.0
```

Comment:
You can also do `ghcup install cabal latest`. Unless you want to track cabal releases.

Reply:
OK, thank you @hasufell